modules/button/react/README.md

```diff
@@ -27,7 +14,43 @@ or
 yarn add @workday/canvas-kit-react-button
 ```
 
-## Usage
+---
+
+## Deprecation Warning
+
+> We are rolling out our next iteration of our buttons and will be deprecating the old style (orange
+> primary, and accompanying secondary, and delete). These are still avialable, but will be removed
+> in the first major release after they are available for all Workday customers. The biggest change
+> is with regards to colors and styling, but the behavior should remain the same.
+
+### New Button
+
+Anywhere you were using `Button`, you will automatically get the updated styling (previously
+`beta_Button`). This will be a visual breaking change (padding and colors have changed). Note, we
+are still supporting the import for `beta_Button` as well. The new buttons include: blue primary
+button, and accompanying secondary, delete, outline, and dropdown buttons. The import and usage is
+documented below.
```

Comment:
It would be nice to mention somewhere in here that if you're using `import { beta_Button as Button } ...` you can remove that now too. Because this too should be removed in a future release.

Reply:
👍